src/main/pages/setup-docker/docker-single-user.adoc

```diff
@@ -157,123 +161,130 @@ docker run -it --rm -v /var/run/docker.sock:/var/run/docker.sock
-
-1.  Start Che In Offline Mode
-
-Place the TAR files into a folder in the offline computer. If the files are in placed in a folder named `/tmp/offline`, you can run Che in offline mode with:
-
++
+The CLI downloads images and saves them to the `/backup/*.tar` file with each image saved as its own file. You can save these files at a different location by volume mounting a local folder to the `:/data/backup` file. The version tag of the CLI Docker image is used to determine the versions of dependent images that must be downloaded. Approximately, one GB of data is saved.
++
+The default execution does not download any optional stack images that are needed to launch workspaces of a particular type. There are many stacks for different programming languages and some of them are over one GB in size. It is unlikely that your users will need all the stacks. So, you do not have to download all of them. For a list of available stack images, run the `eclipse/che offline --list` command. To download a specific stack, run the `eclipse/che offline --image:<image-name>` command. You can use the `--image` flag repeatedly on a single command line.
++
+. Start Che in the offline mode.
++
+Place the TAR files in a folder in the offline computer. If the files are placed in a folder named `/tmp/offline`, run Che in the offline mode using the following command:
++
 ----
 # Load the CLI
 docker load < /tmp/offline/eclipse_che:<version>.tar
 
 # Start Che in offline mode
 docker run <other-properties> -v /tmp/offline:/data/backup eclipse/che:<version> start --offline
 ----
-
-The `--offline` parameter instructs the Che CLI to load all of the TAR files located in the folder mounted to `/data/backup`. These images will then be used instead of routing out to the Internet to check for DockerHub. The preboot sequence takes place before any CLI functions make use of Docker. The `eclipse/che start`, `eclipse/che download`, and `eclipse/che init` commands support `--offline` mode which triggers this preboot sequence.
++
+The `--offline` parameter instructs the Che CLI to load all the TAR files located in the folder mounted to the `/data/backup` file. These images are then used instead of routing out to the Internet to check for DockerHub. The preboot sequence takes place before any CLI functions make use of Docker. The `eclipse/che start`, `eclipse/che download`, and `eclipse/che init` commands support the `--offline` mode that triggers this preboot sequence.
```

Comment:
* -> to the `/data/backup/` directory
* routing out -> routing
* DockerHub -> Docker Hub (also in other places)
* make use of -> use (don't use phrasal verbs)
* Docker -> the Docker daemon
* the `--offline` mode that -> the `--offline` parameter, which